frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -215,6 +215,7 @@ class Slider extends React.PureComponent<Props, State> {
   private renderThumb = React.forwardRef<HTMLDivElement, SharedProps>(
     (props: SharedProps, ref): JSX.Element => {
       const { $value, $thumbIndex } = props
+      // @ts-ignore
```

Comment:
what is the underlying ts complain here?

Reply:
There are two complaints there:
* > TS2538: Type 'undefined' cannot be used as an index type.
* > TS2532: Object is possibly 'undefined'.

The tricky thing is when I put `if ($value && $thumbIndex)` before the line, in some cases the logic broke 🤔 
Thus I've just ignored the warning and this resulted in expected behaviour 🐈‍⬛ 

I've updated the PR with more elegant solution, which avoids `// @ts-ignore`:

```
const formattedValue = $value
        ? this.formatValue($value[$thumbIndex as number])
        : ""
```